### PR TITLE
Define JSON field definitions for Zambia

### PR DESCRIPTION
**Pull request by GPaoloni**
Jira: https://bugs.benetech.org/browse/CHI-324
Specs: https://benetech.app.box.com/file/732932930346

### CODE REVIEW

**Review by andvirga — CHANGES_REQUESTED** [review]
The ZM Screen Requirements document has some specific values for Channel dropdown: 
```
Telephone
Text message (SMS)
Facebook Messenger
Twitter
Webchat
WhatsApp chat
Bulletin Board
E-mail
Mobile app
Outreach
Post
Walk-in / In-person
Website forum
Other
```

However, the options shown in the dropdown are slightly different:
image

File: plugin-hrm-form/src/formDefinitions/tabbedForms/CallerInformationTab.json [hunk 1]
```diff
@@ -2,203 +2,260 @@
...
   },
   {
     "name": "age",
     "label": "Age",
     "type": "select",
     "options": [
-      { "value": "", "label": "" },
-      { "value": "0-03", "label": "0-03" }, 
-      { "value": "04-06", "label": "04-06" }, 
+      { "value": "Unknown", "label": "" },
```
Comment (andvirga) [inline plugin-hrm-form/src/formDefinitions/tabbedForms/CallerInformationTab.json:230]: Per ZM Screen Requirements doc, age should be an incremental dropdown field with values from 0 - 25 and also with ">25" and "Unknown" option.

File: plugin-hrm-form/src/formDefinitions/tabbedForms/ChildInformationTab.json [hunk 2]
```diff
@@ -2,165 +2,251 @@
...
-      { "value": "13-15", "label": "13-15" }, 
-      { "value": "16-17", "label": "16-17" },
-      { "value": "18-25", "label": "18-25" }, 
-      { "value": ">25", "label": ">25" }, 
-      { "value": "Unknown", "label": "Unknown" }
-    ],
-    "required": { "value": true, "message": "RequiredFieldError" }
+    "name": "phone2",
+    "label": "Phone #2",
+    "type": "input"
```
Comment (andvirga) [inline plugin-hrm-form/src/formDefinitions/tabbedForms/ChildInformationTab.json:229]: Per document: "Phone numbers should be numeric only but should probably allow for + and for spaces", so I'm not sure if we can support that yet. Maybe with a RegEx expression?

File: plugin-hrm-form/src/formDefinitions/tabbedForms/ChildInformationTab.json [hunk 3]
```diff
@@ -207,31 +288,57 @@
...
+    "type": "mixed-checkbox"
+  },
+  {
+    "name": "livingInPoverty",
+    "label": "Child living in poverty",
+    "type": "mixed-checkbox"
   },
   {
     "name": "memberOfAnEthnic",
     "label": "Child member of an ethnic / racial minority",
```
Comment (andvirga) [inline plugin-hrm-form/src/formDefinitions/tabbedForms/ChildInformationTab.json:305]: This should state "Child member of an ethnic, racial or **religious** minority"

File: plugin-hrm-form/src/formDefinitions/tabbedForms/ChildInformationTab.json [hunk 4]
```diff
@@ -207,31 +288,57 @@
...
+    "type": "mixed-checkbox"
+  },
+  {
+    "name": "outOfSchool",
+    "label": "Out-of-school child",
+    "type": "mixed-checkbox"
+  },
+  {
+    "name": "childOnTheMove",
+    "label": "Child on the move / in migration",
```
Comment (andvirga) [inline plugin-hrm-form/src/formDefinitions/tabbedForms/ChildInformationTab.json:325]: This should be only "Child on the move"

File: plugin-hrm-form/src/formDefinitions/tabbedForms/CallerInformationTab.json (hunk 1 above)
Comment (GPaoloni) [reply plugin-hrm-form/src/formDefinitions/tabbedForms/CallerInformationTab.json:230]: That's a separate taks, not included in this PR as it involves some changes in chatbot related code.
https://bugs.benetech.org/browse/CHI-405

File: plugin-hrm-form/src/formDefinitions/tabbedForms/ChildInformationTab.json (hunk 2 above)
Comment (GPaoloni) [reply plugin-hrm-form/src/formDefinitions/tabbedForms/ChildInformationTab.json:229]: This one we agreed with @nickhurlburt  that will be a separate PR aswell, as I need to implement a new input type that is not part of the customization yet!

**Comment by GPaoloni** [comment]
@nickhurlburt About @andvirga 's comment above (the channel options), we are now bundling those as a static file in the code. I can implement that to be dynamic aswell, but will do that when we start moving all the definitions to live in an API, and will bundle the ZM channels in this version. Is that right?

**Review by andvirga — APPROVED** [review]
Thank you for addressing the changes. This looks very good!

**Pull request merged**